src/settings.rs

```diff
@@ -36,6 +36,18 @@ pub fn read_policies_file(path: &Path) -> Result<HashMap<String, Policy>> {
     Ok(ps)
 }
 
+#[derive(Deserialize, Debug, Clone)]
+pub struct VerificationSettings {
+    pub verification_keys: HashMap<String, String>,
```

Comment:
I agree, but I would prefer not do do it now. Mainly, because I have no way to test it's working, and adding it in the future is backwards compatible (just mark `verification_keys` as optional in verification.yml with an Option).

(same for the other comments on preparing for keyless mode).